[aoti] Remove example inputs from aoti_compile_and_package

angelayi opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #140991
* #140990



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang @aakhundov

Differential Revision: [D66136724](https://our.internmc.facebook.com/intern/diff/D66136724)

angelayi committed:
[aoti] Remove example inputs from aoti_compile_and_package

[ghstack-poisoned]

angelayi commented:
@angelayi has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D66136724).

yushangdi reviewed (approved):
LGTM

desertfire commented on torch/_inductor/__init__.py:
```diff
@@ -44,8 +48,8 @@ def compile(
-    args: Tuple[Any],
-    kwargs: Optional[Dict[str, Any]] = None,
+    _deprecated_unused_args=None,
+    _deprecated_unused_kwargs=None,
```
These were not announced in 2.5, so I feel it is ok to simply drop them. You can do that as a follow-up PR and keep your warnings run for a while for now, as long as the change lands before the 2.6 branch cut date.  There are some internal references need to be updated when you drop these parameters.

angelayi committed:
Update on "[aoti] Remove example inputs from aoti_compile_and_package"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang aakhundov

Differential Revision: [D66136724](https://our.internmc.facebook.com/intern/diff/D66136724)

[ghstack-poisoned]

angelayi commented:
@angelayi has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D66136724).

deependujha commented:
@angelayi going through the [torch aotinductor c++](https://pytorch.org/docs/main/torch.compiler_aot_inductor.html) docs (main branch, not stable). It gives `segmentation fault` when trying to load `torch._inductor.aoti_compile_and_package` in c++. 

To use the latest, I downloaded pytorch nightly, and libtorch nightly. I'm using Lightning AI's L4 GPU machine.

Any suggestion on what can I do to make it work?

angelayi commented:
@deependujha Can you please make an issue with your repro.

huydhn commented:
@angelayi Are you planning to update ExecuTorch too after this change?  I'm seeing this is used there https://github.com/pytorch/pytorch/actions/runs/12110442973/job/33761234395#step:22:7060
